src/screens/GroupSetting/hooks/useGroupSetting.js

```diff
@@ -41,7 +42,7 @@ const useGroupSetting = ({navigation, route}) => {
     if (changeImage) {
       if (withLoading) setIsLoading(true);
       try {
-        const res = await uploadFile(`data:image/jpeg;base64,${base64Profile}`);
+        const res = await ImageUtils.uploadImage(urlImage);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change is significant because it alters the way images are uploaded. Previously, the image was uploaded as a base64 string, but now it's being uploaded using a URL. This could potentially improve performance by reducing the amount of data that needs to be sent over the network. However, it's important to ensure that the server can handle this new format and that the URL is secure.
```diff
- const res = await uploadFile(`data:image/jpeg;base64,${base64Profile}`);
+ const res = await ImageUtils.uploadImage(urlImage);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->